openfisca_core/tracers.py

```diff
@@ -130,53 +162,125 @@ def _get_time_in_sec(self) -> float:
     def print_computation_log(self, aggregate = False):
         self.computation_log.print_log(aggregate)
 
-    def generate_performance_graph(self, dir_path):
+    def generate_performance_graph(self, dir_path: str) -> None:
         self.performance_log.generate_graph(dir_path)
 
+    def generate_performance_tables(self, dir_path: str) -> None:
+        flat_trace = self.get_flat_trace()
+
+        csv_rows = [
+            {'name': key, 'calculation_time': trace['calculation_time'], 'formula_time': trace['formula_time']}
+            for key, trace in flat_trace.items()
+            ]
+        self._write_csv(os.path.join(dir_path, 'performance_table.csv'), csv_rows)
+
+        aggregated_csv_rows = [
+            {'name': key, **aggregated_time}
+            for key, aggregated_time in self.aggregate_calculation_times(flat_trace).items()
+            ]
+
+        self._write_csv(os.path.join(dir_path, 'aggregated_performance_table.csv'), aggregated_csv_rows)
+
+    def generate_performance_data(self, dir_path: str) -> None:
```

Comment:
I'm not sure it's a good idea to tie these two tasks together, as the output format is very different and will likely be of interest in different circumstances.

Before this PR, we generate the HTML visualization by invoking the test runner with the` --performance` option. This is unambiguous.

After this PR, the `--performance` option does two things, and possibly we're going to be interested in only one of them. It would seem more useful to me to have either two variants (e.g. `--performance:graph` and `--performance:tables`) or two entirely distinct commands.

Reply:
👌, split the 2 options.